### PR TITLE
FAT-17086: Fix mod-search tests.

### PR DESCRIPTION
**Pull request by SvitlanaKovalova1**
## Purpose
[FAT-17086](https://folio-org.atlassian.net/browse/FAT-17086) Fix mod-search tests

### CODE REVIEW

File: mod-search/src/test/resources/spitfire/mod-search/browse/contributor-browse.feature [hunk 1]
```diff
@@ -290,6 +290,15 @@ Feature: Tests that browse by contributors
...
       },
+      {
+        "isAnchor": false,
+        "totalRecords": 1,
+        "contributorTypeId": [
+          "6e09d47d-95e2-4d8a-831b-f777b8ef6d81"
+        ],
+        "name": "Brie.",
+        "contributorNameTypeId": "e8b311a6-3b21-43f2-a269-dd9310cb2d0a"
+      },
```
Comment (psmagin) [inline mod-search/src/test/resources/spitfire/mod-search/browse/contributor-browse.feature:301]: There shouldn't be 2 same values in the response. Probably there is some bug on mod-search side. Please check it

File: mod-search/src/test/resources/spitfire/mod-search/set-up/add-okapi-permissions.feature [hunk 2]
```diff
@@ -0,0 +1,74 @@
+Feature: prepare data for api test
```
Comment (psmagin) [inline mod-search/src/test/resources/spitfire/mod-search/set-up/add-okapi-permissions.feature:1]: Why do we need this?

Comment (SvitlanaKovalova1) [reply mod-search/src/test/resources/spitfire/mod-search/set-up/add-okapi-permissions.feature:1]: Without these permissions it was not possible to run tests locally in the development environment. Perhaps they are only needed for local running.

File: mod-search/src/test/resources/spitfire/mod-search/browse/contributor-browse.feature (hunk 1 above)
Comment (SvitlanaKovalova1) [reply mod-search/src/test/resources/spitfire/mod-search/browse/contributor-browse.feature:301]: They are not the same, they have different names:
"Brie."
"Brie"
"brie."

File: mod-search/src/test/resources/karate-config.js [hunk 3]
```diff
@@ -93,6 +93,14 @@ function fn() {
     }
     config.prototypeTenant = '${prototypeTenant}';
     karate.configure('ssl',true);
+  } else if (env == 'dev') {
+    config.baseUrl = 'https://folio-dev-spitfire-okapi.ci.folio.org';
```
[the remark this answers is not in the document]
Comment (SvitlanaKovalova1) [reply mod-search/src/test/resources/karate-config.js:83]: done

**Review by psmagin — APPROVED** [review]

**Review by mukhiddin-yusuf — APPROVED** [review]

**Pull request merged**